### ISSUE TITLE
Plans: Unable to view plan details, non-functioning Contact Support button

### ISSUE THREAD

**Issue by hoverduck** [wordpress-mobile/WordPress-iOS#6480]
### Expected behavior
To be able to view the details of the WordPress.com plans

### Actual behavior
Opening the details for any plan results in an "Oops There was an error loading the plan" message with a button labeled "Contact support" that does nothing when tapped.  Tested on multiple accounts.

![803111563_8222298319104245285](https://cloud.githubusercontent.com/assets/7233112/22345312/ac8be6c4-e3cd-11e6-9b64-206cb3ac9c46.jpg)

**Comment by koke** [comment]
What's failing is that there are a few plan features that don't have an icon, and the plans remote is failing with a decoding error:

```
(lldb) po json.filter({ $0["icon"] as? String == "" }).map({ $0["title"]! })
▿ 8 elements
  - 0 : Google Analytics
  - 1 : Spam protection
  - 2 : Off-site backups
  - 3 : Backup archive
  - 4 : Backup storage space
  - 5 : Automated Restores
  - 6 : Malware Scanning
  - 7 : Polls and Ratings
```

@frosty what do you think? Should we show those features without icon or ignore them? Definitely not fail completely.

Also looking into why that useless "Contact support" appears

**Comment by koke** [comment]
This was fixed in #6484
